packages/parser/test/integration/positiveTestsHelper.ts

```diff
@@ -265,6 +265,15 @@ export const ExpressionsTests = [
     `,
     output: `{\"type\":\"Program\",\"body\":{\"type\":\"InitStatement\",\"body\":[{\"type\":\"ExpressionStatement\",\"expression\":{\"type\":\"AssignmentExpression\",\"operator\":\"*=\",\"left\":{\"type\":\"IdentifierExpression\",\"name\":\"a\"},\"right\":{\"type\":\"NumericLiteral\",\"value\":4}}}]}}`,
   },
+  {
```

Comment:
There are no invalid tests added for value like : 789....456 . Your regex is failing for these values. Please think through all of the test cases carefully.